posthog/session_recordings/ai/generate_embeddings.py

```diff
@@ -0,0 +1,58 @@
+from openai import OpenAI
+
+from posthog.api.activity_log import ServerTimingsGathered
+from posthog.utils import get_instance_region
+from posthog.models import User, Team
+
+from posthog.session_recordings.models.session_recording import SessionRecording
+from posthog.session_recordings.queries.session_replay_events import SessionReplayEvents
+from posthog.session_recordings.ai.utils import (
+    SessionSummaryPromptData,
+    reduce_elements_chain,
+    collapse_sequence_of_events,
+)
+
+
+def generate_team_embeddings(team: Team):
+    recordings = []  # TODO: figure out how to find unembedded recordings
```

Comment:
Largely depends on the above decision around where to store the embeddings but would love some help (possibly async if someone fancies adding a few commits to this PR) around writing a query to find recordings. In my mind we need to lookup recordings:
- by team
- where the embeddings column is empty
- within the last week